voip_utils/sip.py

```diff
@@ -48,6 +49,71 @@ def caller_rtcp_port(self) -> int:
         return self.caller_rtp_port + 1
 
 
+@dataclass
+class RtpInfo:
+    """Information about the RTP transport used for the call audio."""
+
+    rtp_ip: str | None
+    rtp_port: int | None
+    payload_type: int | None
+
+
+@dataclass
+class SipEndpoint:
```

Comment:
I think after adding the URI as the device identifier it will probably make more sense to construct this with the URI and then parse out the individual parts rather than building the URI from parts like it is doing now.